test/integration/targets/get_certificate/tasks/main.yml

```diff
@@ -0,0 +1,2 @@
+- name: Download Cert as pem from server
+  shell: "python targets/get_certificate/files/process_certs.py {{ httpbin_host }} {{ output_dir }}/temp.pem"
```

Comment:
You need an `- include_tasks: ../tests/validate.yml`, otherwise the tests in `tests/validate.yml` are never executed. (Compare [here](https://github.com/ansible/ansible/blob/devel/test/integration/targets/openssl_csr/tasks/main.yml#L69).)